src/database/seed.sql

```diff
@@ -0,0 +1,10 @@
+INSERT INTO todos (description) VALUES
+('Groceries'),
+('Clean Room'),
+('Meditate'),
+('Jobs')
+;
+INSERT INTO todos (description, is_completed) VALUES
+('Get Food', true),
+('Cook', true)
+;
```

Comment:
since the table defaults the is_completed to true, you do not need to specify this here.